.github/workflows/issue_triage.yml

```
@@ -0,0 +1,88 @@
+name: Issue Triage and Management
+
+on:
+  issues:
+    types: [opened, labeled, assigned]
+
+permissions:
+  issues: write
+
+jobs:
+  triage-on-issue-open:
+    runs-on: ubuntu-latest
+    if: github.event.sender.login != 'github-actions'
+    steps:
+      - name: Reply to Issue (Only on Opened)
```

```suggestion
      - name: Reply to Issue
```
I think the original name was ok. The developer can understand "Only on Opened" from this line `if: ${{ github.event.action == 'opened' }}`